src/App.js

```diff
@@ -1,34 +1,15 @@
 import logo from "./logo.svg";
 import "./App.css";
-const todoList = [
-  {
-    id: 1,
-    title: "My First Assignment in CODE THE Dream.",
-  },
-  {
-    id: 2,
-    title: "I will be try to complete Assigment.",
-  },
-  {
-    id: 3,
-    title: "Finally I am completed my assignment.",
-  },
-  {
-    id: 4,
-    title: "LOVE YOU ALL CODE THE DREAM.",
-  },
-];
+import TodoList from "./components/TodoList";
+import AddTodoForm from "./components/AddTodoForm";
+import { Fragment } from "react";
+
 function App() {
   return (
-    <>
-      <h1>My First Assigment on CODE THE DREAM</h1>
-      {/** Render the items */}
-      <ul>
-        {todoList.map((item) => {
-          return <li key={item.id}>{item.title}</li>;
-        })}
-      </ul>
-    </>
+    <Fragment>
+      <TodoList />
+      <AddTodoForm />
+    </Fragment>
```

Comment:
It's entirely up to you, but common convention is to just use `<></>` for for [document fragments in React](https://reactjs.org/docs/fragments.html#short-syntax). Used this way in a Create React App (CRA) application, you do not need to import it either.